cypress/e2e/po/components/sortable-table.po.ts

```diff
@@ -55,6 +55,18 @@ export default class SortableTablePo extends ComponentPo {
     return cy.getId('sortable-table-promptRemove');
   }
 
+  deactivateButton() {
+    return cy.getId('sortable-table-deactivate');
```

Comment:
Is this one needed? I couldn't the data-testid in code. It also looks list specific, rather than one available to all lists

Reply:
Its a dynamic data-testid which comes from https://github.com/rancher/dashboard/blob/7896059c49655f54187749144db9cd68792d0ecd/shell/components/SortableTable/index.vue#L935, but yes I think you're right it seems to be list specific. I'll move it to users.po.ts